Automattic/wp-calypso
# Hosting Command Palette: Trim double quotes from selected items if present

Related to https://github.com/Automattic/dotcom-forge/issues/4550

## Proposed Changes

This PR trims the double quotes from the selected site title before it is processed by the `document.querySelector`. This ensures that the command palette does not break if the site titles include double quotes.

This is what the error looks like if there are double quotes present before they are being processed by the `document.querySelector` in the command palette: 

<img width="1292" alt="Screenshot 2023-11-21 at 3 16 33 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/6711dd1e-2801-4341-868a-f959480a8ee2">

## Testing Instructions

* Ensure that you have an Atomic site
* Set the site title under **Settings > General** on an Atomic site to include double quotes e.g. "My Site"
* First, test on `https://wpcalypso.wordpress.com/sites` by triggering `cmd + k` on Mac and `ctrl + k` on Windows
* Navigate to one of the SSH commands e.g. `Open SSH details`
* Try navigating to the site with double quotes in the site title
* Confirm this breaks the command palette
* Next, navigate to Calypso Live Link generated in this PR
* Trigger command palette with `cmd + k` on Mac and `ctrl + k` on Windows
* Try navigating to the site with double quotes in the site title
* Confirm that the command executes as expected 

## Pre-merge Checklist
